aosp_diff/preliminary/build/make/0004-Update-security_patch_level-string.patch

```diff
@@ -20,7 +20,7 @@ index 339d36bb5c..85b5300b2d 100644
      #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
      #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
 -    PLATFORM_SECURITY_PATCH := 2023-02-05
-+    PLATFORM_SECURITY_PATCH := 2023-04-01
++    PLATFORM_SECURITY_PATCH := 2023-05-01
```

Comment:
Change the string only after concluding on the left out patches.

Reply:
Left patches are added